ballerina/tests/resources/documents/intersection_types.graphql

```diff
@@ -0,0 +1,77 @@
+query getName {
+  name(species: { genus: "Craspedocephalus", specificName: "trigonocephalus" })
```

Comment:
```suggestion
  name(species: {genus: "Craspedocephalus", specificName: "trigonocephalus"})
```
is this the correct formatting here? Seems like starting and ending spaces are added automatically.

Reply:
Yes, the GraphQL formatter adds the spaces.